config/ignition_fortress_debian_buster.yaml

```diff
@@ -0,0 +1,96 @@
+name: ignition_fortress_debian_buster
+method: http://packages.osrfoundation.org/gazebo/debian-stable
+suites: [buster]
```

Comment:
Rolling is going to switch to Debian Bullseye alongside Ubuntu Jammy. Are we currently building Fortress for Bullseye?

Reply:
Not yet, see https://github.com/ignition-tooling/release-tools/issues/566